### DISCUSSION TITLE
Is it possible to add a pop up link in the custom text?

### DISCUSSION THREAD

**Discussion by FuFuMaxime** [globaleaks/globaleaks-whistleblowing-software#4284] [Q&A]
Hi,

I have a question about customizable text, I'd like to know if it's possible to add a link who open a little popup with a texte ?

I know that we can add a clickable link with this exemple :

Click here to open [google](https://www.google.com)

Maybe this feature already exists but I haven't found how to do it

Thanks for the response

**Comment by maelaborazioni** [comment]
yes, if you use javascript...
